Libraries/PeriphDrivers/Source/SYS/sys_me21.c

```diff
@@ -160,6 +163,8 @@ int MXC_SYS_RTCClockDisable(void)
     /* Check that the RTC is not the system clock source */
     if ((MXC_GCR->clkctrl & MXC_F_GCR_CLKCTRL_SYSCLK_SEL) != MXC_S_GCR_CLKCTRL_SYSCLK_SEL_ERTCO) {
         MXC_GCR->clkctrl &= ~MXC_F_MCR_CLKCTRL_ERTCO_EN;
```

Comment:
This may be an error in the UG and not in the code, but should this be clearing a bit in MCR? Or should there be an added field in GCR->clkctrl?

Reply:
Good catch!  Just pushed a change that writes to the MXC_MCR->clkctrl instead of MSC_GCR->clkctrl.  Luckily bit 17 is undefined in GCR, so the old code wasn't doing anything bad.